### ISSUE TITLE
crash

### ISSUE THREAD

**Issue by wasilev** [codesandbox/codesandbox-client#3462]
<h1>💥 Crash Report</h1>

  <h2>What were you trying to accomplish when the crash occurred?</h2>

  > Please use this issue template to describe what you were doing when you encountered this crash. While we are able to fill in some details automatically, it's not always enough to reproduce!

  <h3>Link to sandbox: [link]() (optional)</h3>

  <h3>Crash Details</h3>

  <details>
  <summary>Environment</summary>

  | Browser |  Version  | Operating System | CodeSandbox Version |
  | ------- | --------- | ---------------- | ------------------- |
  | chrome | 79.0.3945 | Windows 10           | PROD-1580767480-c435a7ba4          |

  **Route:**
  https://codesandbox.io/s/n72e7

  </details>

  <details>
  <summary>Error Message</summary>

  ```bash
  TypeError: Cannot read property 'username' of null
  TypeError: Cannot read property 'username' of null
    at ie (https://codesandbox.io/static/js/app.df1b562ec.js?_sw-precache=54fc394ac461c1534115c6ac4e5c77ff:532:1407274)
    at da (https://codesandbox.io/static/js/common.200e59a13.chunk.js?_sw-precache=8275c48d07f50ea6d3ce1a7d80d3c47b:1:1695375)
    at $a (https://codesandbox.io/static/js/common.200e59a13.chunk.js?_sw-precache=8275c48d07f50ea6d3ce1a7d80d3c47b:1:1703618)
    at Ss (https://codesandbox.io/static/js/common.200e59a13.chunk.js?_sw-precache=8275c48d07f50ea6d3ce1a7d80d3c47b:1:1741961)
    at Sl (https://codesandbox.io/static/js/common.200e59a13.chunk.js?_sw-precache=8275c48d07f50ea6d3ce1a7d80d3c47b:1:1726418)
    at xl (https://codesandbox.io/static/js/common.200e59a13.chunk.js?_sw-precache=8275c48d07f50ea6d3ce1a7d80d3c47b:1:1726343)
    at gl (https://codesandbox.io/static/js/common.200e59a13.chunk.js?_sw-precache=8275c48d07f50ea6d3ce1a7d80d3c47b:1:1723663)
    at https://codesandbox.io/static/js/common.200e59a13.chunk.js?_sw-precache=8275c48d07f50ea6d3ce1a7d80d3c47b:1:1681659
    at https://codesandbox.io/static/js/common.200e59a13.chunk.js?_sw-precache=8275c48d07f50ea6d3ce1a7d80d3c47b:1:1757109
    at Go (https://codesandbox.io/static/js/common.200e59a13.chunk.js?_sw-precache=8275c48d07f50ea6d3ce1a7d80d3c47b:1:1681369)
  
    in ie
    in div
    in t
    in elements__UserMenuContainer
    in div
    in t
    in elements__AccountContainer
    in div
    in t
    in elements__Right
    in header
    in t
    in elements__Container
    in Lr
    in div
    in t
    in elements__Container
    in t
    in Unknown
    in eh
    in t
    in t
    in div
    in t
    in elements__Content
    in t
    in withRouter(t)
    in div
    in t
    in elements__Container
    in Unknown
    in withRouter()
    in t
    in Unknown
    in t
    in l
    in t
  ```

  </details>

**Comment by garethx** [comment]
@wasilev What were you trying to accomplish when the crash occurred?

**Comment by lbogdan** [comment]
`user` is `null` at
https://github.com/codesandbox/codesandbox-client/blob/d325b7206705fd5bc1896617c7f6cbabb3ccce0a/packages/app/src/app/pages/common/UserMenu/index.tsx#L30-L35

**Comment by lbogdan** [comment]
Closing this, it should already by fixed by #3659 .
